pyproject.toml

```diff
@@ -37,6 +37,11 @@ grpcio = ">=1.56.0"
 plotly = "^5.15.0"
 pyarrow = ">=4.0.0"
 grpcio-status = ">=1.56.0"
+lib = "^4.0.0"
```

Comment:
what is this about?

Reply:
ah I think this one is not needed anymore, I'll update the dependencies